src/SelfService/Domain/Models/ECRRepositoryId.cs

```diff
@@ -0,0 +1,56 @@
+namespace SelfService.Domain.Models;
+
+public class ECRRepositoryId : ValueObject
```

Comment:
Why is our 'id' a 'ValueObject'.
This is DDD gone wrong, but oh well -- this is not a criticism about this code, but of our general approach which we now can only follow, if we do not yank everything up